@xen-orchestra/rest-api/src/groups/group.controller.mts

```diff
@@ -49,4 +70,25 @@ export class GroupController extends XoController<XoGroup> {
   getGroup(@Path() id: string): Promise<Unbrand<XoGroup>> {
     return this.getObject(id as XoGroup['id'])
   }
+
+  /**
+   * @example body {
+   * "name": "new group",
+   * "provider": "64a7a0b4-e728-47e2-a082-93a218890a81",
+   * "providerGroupId": "722d17b9-699b-49d2-8193-be1ac573d3de"
+   * }
+   */
+  @Example(groupId)
+  @Post('')
+  @Middlewares(json())
+  @SuccessResponse(createdResp.status, createdResp.description)
+  @Response(resourceAlreadyExists.status, resourceAlreadyExists.description)
+  @Response(invalidParameters.status, invalidParameters.description)
+  async createGroup(
+    @Body() body: { name: string; provider?: string; providerGroupId?: string }
```

Comment:
If we compare with the `JSON RPC-API`, `group.create` only take a `name`  parameter